### PR TITLE
Passing mapping from sr_rhand.launch sr_lhand.launch

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

**Comment by ugocupcic** [comment]
Goes with https://github.com/shadow-robot/sr-config/pull/51

**Comment by ugocupcic** [comment]
Tested on @toliver hand

File: sr_robot_lib/launch/sr_hand_lib.launch [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+  <!-- Defines the sub-directory used in
        $(find sr_ethercat_hand_config)/mappings/
-       and $(find sr_ethercat_hand_config)/calibrations/ 
+       and $(find sr_ethercat_hand_config)/calibrations/
        to know where to find the parameters for a certain hand.
-       In case we only have one hand this parameter will normally be "", meaning that the desired params are in the root of those directories  
+       In case we only have one hand this parameter will normally be "", meaning that the desired params are in the root of those directories
        If it's not "" then it must be followed by a "/" e.g. "hand_2/"  -->
-  <arg name="hand_id" default=""/>
+  <arg name="hand_id" default="rh/"/>
```
Comment (toliver) [inline sr_robot_lib/launch/sr_hand_lib.launch]: We are using "rh" instad of "rh/" in all the launchfiles now.

File: sr_edc_launch/load_hand_parameters.xml [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <launch>
   <!-- The id of the hand will be used to namespace the hand parameters
        the id should   -->
-  <arg name="hand_id" default=""/>
+  <arg name="hand_id" default="rh/"/>
```
Comment (toliver) [inline sr_edc_launch/load_hand_parameters.xml]: Same here.

**Comment by toliver** [comment]
Happy to merge after addressing those

**Pull request merged**